gapic-common/lib/gapic/paged_enumerable.rb

```diff
@@ -95,7 +95,7 @@ def each
     #
     # @yield [Page] Gives the pages in the stream.
     #
-    # @raise [GapicError] if it's not started yet.
+    # @raise if it's not started yet.
```

Comment:
I don't know where this error is coming from. I think it might have been copied over from Gax, but I dunno. I'm not sure what type of error this is, if it even exists.

Reply:
Ah, if it was already wrong and we don't know maybe put another `TODO` so we catch it during the doc pass? It would be good to document the error types since this is used for many things.